test/extended/testdata/builds/test-build-app/utf8_tést_app.rb

```diff
@@ -0,0 +1,7 @@
+require 'rack'
+ 
+app = Proc.new do |env|
+    ['200', {'Content-Type' => 'text/html'}, ['Wë súpport UTF-8!']]
```

Comment:
@adambkaplan this has uncovered a bug in our downstream tools. We use tito to build the RPM from source, and apparently it's never had to handle non-ascii in a filename before. It breaks. So I'm evaluating best course of action. Fixing tito may be hard. Deleting this file is easy. If I delete this file in the private repo, am I going to run into a boatload of merge conflicts down the road? Or will this never change? Would this test be just as effective with an ascii filename? Just evaluating alternatives, please opine.

Reply:
also you should have put an emoji in there just to visually distinguish from latin-1 :)

Reply:
@sosiouxme This test won't be as effective, as the non-ascii file name verifies a bug fix in a black-box fashion. It's a trivial bug, so I'm OK deleting this and adding a separate test.